python/tvm/relay/op/transform.py

```diff
@@ -1322,6 +1322,71 @@ def adv_index(inputs):
     return _make.adv_index(Tuple(inputs))
 
 
+def sparse_fill_empty_rows(sparse_indices, sparse_values, dense_shape, default_value):
```

Comment:
Please document that this takes a sparse matrix in coordinate (COO) format.

Reply:
Done.